[4.1] Theme update: pagefind and assets minification (#5610)
wazuh/wazuh-documentation: CONTRIBUTING.md

```
This will delete the contents from `/wazuh-documentation/build/html`.

## Alternative Pagefind search engine

> To compile the documentation with the alternative Pagefind search engine, you need NodeJS 16+ in your system. Then, replace with `make html-search` the command in step 3 in the previous section and follow all those steps.

Reviewer: ```suggestion
To compile the documentation with the alternative Pagefind search engine, install NodeJS 16+. Then, follow the steps from the previous section replacing with `make html-search` the command in step 3.
```

> To compile the documentation with the alternative Pagefind search engine you need to have NodeJS 16+ in you system and follow the steps listed in the previous section but using `make html-search` instead of `make html` in the step 3.

Reviewer: ```suggestion
To compile the documentation with the alternative Pagefind search engine, you need NodeJS 16+ in your system. Replace with `make html-search` the command in step 3 in the previous section and follow the steps.
```
